axelrod/load_data_.py

```diff
@@ -0,0 +1,45 @@
+import pkg_resources
+
+
+def load_file(filename, directory):
```

Comment:
Yep! I prefer using pandas actually if the extra dependency is ok with @drvinceknight .